content/courses/advanced-ada/chapters/access_types.rst

```diff
@@ -521,7 +521,7 @@ same object as :ada:`I`:
 As we've seen before, we can have multiple calls to :ada:`Free (I)`.
 However, the call to :ada:`Free (I_2)` is bad because :ada:`I_2` is not null.
 In fact, it is a dangling reference |mdash| i.e. :ada:`I_2` points to an object
-that doesn't exist anymore. Also, the previous call to :ada:`Free (I)` will try
+that doesn't exist anymore. Also, the first call to :ada:`Free (I)` will try
```

Comment:
OK, good regarding "first".
One more thing. I should have mentioned this previously. Instead of "will try to reclaim" I think it should be "will reclaim". Then, when talking about Free (I_2) you can say "will try to reclaim the previously-reclaimed object" as the explanation for why the call is bad.

Reply:
Yes, that makes totally sense. I hope I got it right now. :-)